examples/Demo.elm

```diff
@@ -143,66 +233,111 @@ stylesheet = Style.stylesheet """
          inline css.
        */
   }
-  p, blockquote { 
+  p, blockquote {
     max-width: 33em;
     font-size: 13px;
   }
 """
 
 
-
 view : Signal.Address Action -> Model -> Html
 view addr model =
-  let top =
-        div
-          [ style
-            [ ("margin", "auto")
-            , ("padding-left", "5%")
-            , ("padding-right", "5%")
+  routingView addr model
+
+
+routingView : Signal.Address Action -> Model -> Html
+routingView addr model =
+  case (Debug.log "Route " model.routing.route) of
+    Routing.Home ->
+      let
```

Comment:
I don't think the `let` here adds clarity—Elm is verbose enough as it is :). Maybe just a one-liner:

    Routing.Home -> 
      appView addr { model | layout = setTab model.layout 0 }

Same for the next case, `Routing.TabRoute`.